### ISSUE TITLE
Xcode 14.1 with branch 2.4 Target missing

### ISSUE THREAD

**Issue by kkro** [core-plot/core-plot#470]
I think the most importent target is gone. When I use an older coreplot version there is one more. However it does not compile because it generates no libCorePlot-CocoaTouch.a so in the following linker error comes up as expected: 

Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_CPTColor", referenced from:
      objc-class-ref in ViewController.o
  "_OBJC_CLASS_$_CPTPlotRange", referenced from:
      objc-class-ref in ViewController.o
  "_OBJC_CLASS_$_CPTScatterPlot", referenced from:
      objc-class-ref in ViewController.o
  "_OBJC_CLASS_$_CPTTheme", referenced from:
      objc-class-ref in ViewController.o
  "_OBJC_CLASS_$_CPTXYGraph", referenced from:
      objc-class-ref in ViewController.o

Or has something fundamental changed which I do not know?

2.4
<img width="282" alt="Bildschirm­foto 2022-11-18 um 03 26 49" src="https://user-images.githubusercontent.com/11778545/202602774-3a4fd3a4-d630-4f1e-ab59-a26e37ff20d4.png">

older, from 2020

<img width="282" alt="Bildschirm­foto 2022-11-18 um 03 34 41" src="https://user-images.githubusercontent.com/11778545/202603915-25a52282-5468-4006-89fc-75c2f73854b2.png">

**Comment by eskroch** [comment]
The iOS static library has been removed from the `release-2.4` branch. Change your project to refer to the Core Plot iOS framework right above it in the target list instead.

**Comment by myoung959** [comment]
I too get this same set of errors. I have tried for days with every permutation and combination of builds from cocoapods without success. I sure wish that there was a definitive step-by-step set of instructions to help those of us who are not experts on the intricacies of the various settings and conditions within settings needed to get this to work. I have used the static library approach for over 10 years without a hitch, but now just the mantra "install the framework" has bricked my apps because I cannot fathom how to do this. I am beyond frustrated and hope that someone will post an installation tutorial that will work. Thanks.

**Comment by walter-in-space** [comment]
Meanwhile I left coreplot and now I use Swift Charts. I used it for many years but every one or two years the time to make it compile again was to much for me.

**Comment by eskroch** [comment]
If you're using CocoaPods, add the following lines to your pod file:

```ruby
use_frameworks!
pod 'CorePlot', :git => 'https://github.com/core-plot/core-plot.git', :branch => 'release-2.4'
```

If you're linking it directly in your project, click the "+" sign and add it to the "Frameworks, Libraries, and Embedded Content" list on the General tab of the target settings.

<img width="680" alt="Screenshot 2023-04-26 at 7 31 17 PM" src="https://user-images.githubusercontent.com/1934470/234724285-935ce780-bfe2-4e87-b2d3-4e8b68904a34.png">

Go to the "Build Phases" tab of the settings and add Core Plot to the Target Dependencies phase to ensure it gets built before your app.

<img width="585" alt="Screenshot 2023-04-26 at 7 36 02 PM" src="https://user-images.githubusercontent.com/1934470/234724724-a8f38792-b91b-430a-9ebb-6a0c3237bb26.png">

Use the following statement to include the Core Plot headers in your source files:

```obj-c
#import <CorePlot/CorePlot.h>
```

**Comment by myoung959** [comment]
This is hopeless. I have tried for two days to implement the process set forth above, but get nothing but endless errors none of which have explanations that make sense. Errors that say this or that should be deleted ought to be intelligent enough to perform the task for you. When I try to follow the instruction nothing good happens.

Perhaps the sequence above works for Swift apps, because it absolutely doesn't for iOS apps. One seems impossible to execute. Adding CorePlot to the Target Dependencies only seems to allow Swift constructs.

I would be happy to supply the entire source code for my app, which BTW is one of those cited as an example of CorePlot usage, in order to learn how to make this CorePlot framework work. All my CorePlot apps are now bricks because I cannot compile them with the 2.4 version and they won't compile with the newest versions of Xcode and iOS.

**Comment by eskroch** [comment]
I'm happy to take a look at your project to help figure out where the disconnect is. Send me an email (address in my profile) and we can coordinate a way to share the code.

**Comment by eskroch** [comment]
I believe the build problems have been resolved on the `release-2.4` branch. Please let me know if you're still seeing any issues.

**Comment by Mike4U** [comment]
I wonder if you realize that most folks can't get CorePlot to build  on MacOS anymore.  I come back to it every once in a while and no changes.  I'm using your release-2.4 as downloaded with GitHub desktop.

I'm on a MacBook Air M2 with Sonoma 14.0, but this problem arose around 2022.
CorePlot Mac builds with Any Mac(Apple Silicon, Intel) if I set macOS to 10.15.  Unfortunately it only specifies minimum and uses 14.0sdk so I get
NSToolbarSeparatorItem is deprecated since macOS 11.0

When I try Plot Gallery-Mac I get 27 errors that boil down to 
~/core-plot/examples/CorePlotGallery/src/shared/PlotItem.h:8:9: 'CorePlot/CorePlot.h' file not found

Perhaps its something to do with using sdk14.0 even though that would result in lots of errors which are eliminated setting to 10.15
